examples/CalcQt/CMakeLists.txt

```diff
@@ -1,36 +1,14 @@
 project(CalcQt)
 
-set(CALCQT_HEADERS src/CalculatorWidget.h)
-set(CALCQT_SOURCES src/CalcQt.cpp src/CalculatorWidget.cpp)
-include_directories(${CUKE_INCLUDE_DIRS} src)
```

Comment:
I like that you remove this global `include_directories` but you should probably combine it with `target_include_directories(libcalcqt PUBLIC src)` to fix the build errors you get.